### DISCUSSION TITLE
How Do You Use The Porta Proxy Bookmarklet ?

### DISCUSSION THREAD

**Discussion by unrealapex** [QuiteAFancyEmerald/InvisiProxy#43] [Q&A] [answered]
I was wondering how to use the Porta Proxy bookmarklet provided on the bookmarklets page. How do I go to other websites ?

**Comment by QuiteAFancyEmerald** [comment, marked as answer]
Porta Proxy: `javascript:((function(){var a,b,c;c="https://example.com",b=document.createElement("iframe"),b.setAttribute("src",c),b.setAttribute("id","cloak-page"),b.setAttribute("style","position: fixed; width: 100%; height: 100%; top: 0; left: 0; right: 0; bottom: 0; z-index: 99999999999; background-color: #fff;"),a=document.getElementsByTagName("body")[0],a.appendChild(b)})).call(this);`

Edit the part where it says "example.com" to the site you would like to use this on, mostly HU sites. This particular bookmarklet is useful since it bypasses Hapara. Note that not all sites work with this.
